src/RulesTest/RuleRunnerTests.cs

```diff
@@ -341,11 +282,13 @@ public void RunAll_ReturnsExpectedResults()
 
         private static Mock<IRule> CreateRuleMock(Condition condition, EvaluationCode code, A11yElement e)
         {
-            var infoStub = new RuleInfo();
+            var infoStub = new RuleInfo {
+                ErrorCode = code
+        };
```

Comment:
Should be indented one level:
```suggestion
            };
```